docs/reference/setup/upgrade.asciidoc

```diff
@@ -7,6 +7,7 @@ Before upgrading Elasticsearch:
 
 * Consult the <<breaking-changes,breaking changes>> docs.
 * Test upgrades in a dev environment before upgrading your production cluster.
+* Remove your current plugins, then install the latest after upgrading.
```

Comment:
I think this is a good point, can you upgrade to a version of a plugin that you can't run locally before you upgrade?